### DISCUSSION TITLE
rocketmq的5.2.0版本，顺序消费线程数consumeThreadMax问题

### DISCUSSION THREAD

**Discussion by zhl19911203** [apache/rocketmq#8461] [Q&A] [answered]
rocketmq的5.2.0版本，使用rocketmq-spring-boot-starter2.3.0版本，消费者类@RocketMQMessageListener注解如果设置为顺序消费，那么consumeThreadMax和consumeThreadNumber参数需要限制为1吗，还是使用默认值就可以了

**Comment by liuwei178** [comment, marked as answer]
不需要限制，顺序消费是由MQ控制消息的可见性，和consumer的线程没关系的，是两回事

**Reply by zhl19911203** [reply]
好的
